### ISSUE TITLE
Accessibility at touchbar

### ISSUE THREAD

**Issue by christopherpross** [electron/electron#20443]
Greetings,
at the current electron version, I see no way to label controls for a screenreader correctly. I haven't access at the NSAccessibility protocol and the attrebutes for set e.g. a description.

This makes all electron-apps inaccessible for voiceover on the touchbar. This course that the buttons aren't labeled and the visual impered user don't knows, what this button does. The only way to show a text for the voiceover user, is to set the label property. But this label is also print out on the control. In many situations, the developer only wants to use icons. Well, way not to add a binding to the nsaccessibility protocol? Or to the nsaccessibility.ateributes?
see [here](https://developer.apple.com/documentation/appkit/nsaccessibilityattributename?language=objc)

Hopeful you can do this and give the developers the tools to improve the accessibility of apps. Electron is used in many apps, there are could useful for blind users for example vscode or slack.

If you need help or more informations, I will try to help you.

all the best,
Christopher
